Find a functionally equivalent code:android.animation.AnimatorSet.Builder.before:COMMENT
Method Modifier: public      
Comment:/**
 * Sets up the given animation to play when the animation supplied in the
 * {@link AnimatorSet#play(Animator)} call that created this <code>Builder</code> object
 * ends.
 *
 * @param anim The animation that will play when the animation supplied to the
 * {@link AnimatorSet#play(Animator)} method ends.
 */

@@ -1,4 +1,5 @@
 {
+    mReversible = false;
     Node node = mNodeMap.get(anim);
     if (node == null) {
         node = new Node(anim);

